pkgs/applications/version-management/gcli/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, curl
+, autoreconfHook
+, pkg-config
+, yacc
```

Looks like ofBorg is unhappy about yacc (;
```suggestion
, byacc
```